nix/README.md

```diff
@@ -0,0 +1,140 @@
+# YeetMouse NixOS Flake
+
+YeetMouse may be built using this folder's packaged Nix flake.
+As the `flake.nix` file is in the `nix/` folder the flake is accessible under the `github:AndyFilter/YeetMouse?dir=nix` URL.
+
+## NixOS Module Installation
+
+You may install this flake by adding it to your NixOS configuration.
+The following example shows the `inputs.yeetmouse` input and the
+`yeetmouse.nixosModules.default` NixOS module being added to a system
+in your `flake.nix` file.
+    
+```nix
+{
+  # Add this to your flake inputs
+  # Note that `inputs.nixpkgs` assumes that you have an input called
+  # `nixpkgs` and you might need to change it based on your `nixpkgs`
+  # input's name.
+  inputs.yeetmouse = {
+    url = "github:AndyFilter/YeetMouse?dir=nix";
+    inputs.nixpkgs.follows = "nixpkgs";
+  };
+  # <rest of your config> ...
+
+  outputs = { nixpkgs, yeetmouse, ... }: {
+    # This is an example of a NixOS system configuration
+    nixosConfigurations.HOSTNAME = nixpkgs.lib.nixosSystem {
+      system = "x86_64-linux";
+      modules = [
+        # Add the `yeetmouse` input's NixOS Module to your system's modules:
+        yeetmouse.nixosModules.default
+      ];
+    };
+ };
+}
+```
+
+This will expose a new `hardware.yeetmouse` configuration option in your NixOS system's `config`,
+which you can use to activate `yeetmouse`'s device driver, udev rules, and executable.
+
+```nix
+{
+  hardware.yeetmouse = {
+    enable = true;
+    parameters = {
+      # Alter the `parameters` to change the default settings of the `yeetmouse` driver
+      # Check `nix/module.nix` to see all options
+      Acceleration = 1.0;
+    };
+  };
+}
+```
+
+Then, rebuild and switch into your new system (using `nixos-rebuild`). After a reboot, the
+`yeetmouse` driver should be loaded for your connected mouse with the parameters you specified.
+
+After restarting your system, to verify that the driver works start up the yeetmouse GUI:
+
+```sh
+sudo yeetmouse
```

Comment:
```suggestion
sudo -E yeetmouse
```

Reply:
I assume this fixed your issue in that while for Hyprland you configured XWayland-related environment variables in your user's shell configuration, they weren't distributed to a global `environment` configuration and/or provided to the `sudo`'d command?

Reply:
I was thinking about adding the solution to this issue to some kind of *FAQ* section, but this is way cleaner. I'll change the `Readme` to account for this, if it turned out to be an issue on other distros too.

Thank you for that find, I'm very happy the solution turned out to be this simple!

Reply:
To answer @kitten's question: I think so, yes.